rhea/rhea.html

```diff
@@ -309,6 +313,7 @@
           port: { value: '', required: true, validate: RED.validators.number() } ,
           username: { value: '' },
           password: { value: '' },
+          clientId: { value: '' },
```

Comment:
I would use the container-id as from AMQP 1.0 spec instead of changing to client-id.